src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,149 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.time.LocalDateTime;
+
+/**
+ * This class encapsulates the logic of the storage of the task list in the user's harddrive.
+ */
+public class Storage {
+    /* Const fields for representation of the various paths. */
+    private static final String PROJECT_ROOT = System.getProperty("user.dir");
+    private static final Path SAVE_LOCATION = Path.of(PROJECT_ROOT, "data");
+    private static final String SAVE_FILE_NAME = "Task List.txt";
+    private static final Path SAVE_FILE_PATH = (SAVE_LOCATION).resolve(SAVE_FILE_NAME);
+
+    /**
+     * Loads the data from the file found in the saved location into the task list.
+     *
+     * @param tasks task list.
+     */
+    public static void Load(List<Task> tasks) {
```

Comment:
I agree with other commenter, method names should be written in camelCase. I noticed the same issue in your `Save` method too.